src/components/dialogs/form-contingency-dialog.js

```diff
@@ -102,7 +102,9 @@ const FormContingencyDialog = ({ listId, open, onClose, onError, title }) => {
 
     useEffect(() => {
         // get contingency list
-        getCurrentContingencyList(listId);
+        if (listId !== '') {
```

Comment:
needed this change to fix a bug due to change of initialisation of the listId variable. Not needed anymore since it was changed in another PR. Removed.